strapi/strapi
# GenericInput: improve null default-value handling for boolean fields

### What does it do?

Updates the wording for `ToggleCheckbox` fields from `On/Off` to `True/False` and implements a way for clear the field, in case the default value is `null`. It also allows the `ToggleInput` field to display error messages.

~🔥 This builds on top of the design-system `1.0.0` branch.~

### Why is it needed?

Because content editors need a way to reset the field.

### How to test it?

1. Create a boolean field, which a default value of null
2. Update the field value
3. You should be able to see a "Clear" button

### Related issue(s)/PR(s)

- https://strapi-inc.atlassian.net/browse/CONTENT-73
- Refs: https://github.com/strapi/design-system/pull/538
- Refs: https://github.com/strapi/strapi/issues/12372
